helpers/labgrid-raw-interface

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+#
+# Wrapper script to be deployed on machines whose network interfaces should be
+# controllable via the RawNetworkInterfaceDriver. A /etc/labgrid/labgrid.ini
+# can deny access to network interfaces. See below.
+#
+# This is intended to be used via sudo. For example, add via visudo:
+# %developers ALL = NOPASSWD: /usr/local/sbin/labgrid-raw-interface
+
+import argparse
+import os
+import sys
+import configparser
+import warnings
+
+import yaml
+
+
+def get_denylist():
+    try:
+        with open("/etc/labgrid/helpers.yaml") as stream:
+            data = yaml.load(stream, Loader=yaml.SafeLoader)
+    except (PermissionError, FileNotFoundError, AttributeError):
+        warnings.warn("No configuration file (/etc/labgrid/helpers.yaml), inaccessable or invalid yaml, continuing without denylist.")
+        return []
+
+    denylist = data.get("raw-interface", { "denied-interfaces": "lo" }).get("denylist")
+
+    if denylist == [ "lo" ] or not isinstance(denylist, list):
+        warnings.warn("No explicit denied-interfaces or not a list, please check your configuration.")
+        return []
+
+    return denylist
+
+
+def main(program, ifname, capfile, count):
+    if not ifname:
+        raise ValueError("Empty interface name.")
+    if any((c == "/" or c.isspace()) for c in ifname):
+        raise ValueError(f"Interface name '{ifname}' contains invalid characters.")
+    if len(ifname) > 16:
+        raise ValueError(f"Interface name '{ifname}' is too long.")
```

Comment:
Do we need to check if any parameters (ifname, capfile, count) start with `-`? That might confuse tcpdump/tcpreplay...

Reply:
This should not be an issue anymore, since we use longopts now.